tests/openapi_generic_multi_struct.rs

```diff
@@ -39,12 +39,10 @@ fn test_multi_generics_compile() {
     }
```

Comment:
Can you please fix the warning in this file? 
![image](https://user-images.githubusercontent.com/29931815/126944006-e4a0e6e8-9eb1-4e41-844c-6ebfa1d77409.png)

Reply:
sure, my miss!